homeassistant/components/geniushub/__init__.py

```diff
@@ -84,6 +83,9 @@
         return False
     broker.make_debug_log_entries()
 
+    if broker.hub_uid is None:
```

Comment:
I suggest we move this inside the broker class.

Reply:
Done